### PR TITLE
Burn Mechanism Rearrangement

### PR DESCRIPTION
**Pull request by vivek-arte**
This PR rearranges the positioning of the burn mechanism in order to remove duplication of material between the burn mechanism section and the value balance verification section.

### CODE REVIEW

File: zip-0226.rst [hunk 1]
```diff
@@ -159,35 +159,63 @@ The Orchard Protocol uses a Homomorphic Pedersen Commitment [#protocol-concretev
...
+
+The sender includes a :math:`\mathsf{v^{AssetBase}}` variable for every Asset Identifier that is being burnt, which represents the amount of that Asset being burnt. As described in the `ZSA Transaction Structure`_, this is separate from the regular :math:`\mathsf{valueBalance^Orchard}` that is the default transparent value for the ZEC Asset, and represents either the transaction fee, or the amount of ZEC changing pools (eg: to Sapling or Transparent).
+
+For every Custom Asset that is burnt, we add to the :math:`\mathsf{assetBurn}` vector the tuple :math:`(\mathsf{AssetBase},\mathsf{v^{AssetBase}})` such that the validator of the transaction can compute the value commitment with the corresponding value base point of that Asset. This ensures that the values are all balanced out with respect to the Asset Identifiers in the transfer.
+
+.. math:: \mathsf{assetBurn} = \{ (\mathsf{AssetBase},\mathsf{v^{AssetBase}})\ |\ \forall\ \mathsf{AssetBase}\ \textit{s.t.}\ \mathsf{v^{AssetBase}} \neq 0 \}
```
Comment (PaulLaux) [inline zip-0226.rst:171]: Let's use similar notation to the Value commitment definition (and same order):

$(v_{AssetId},AssetBase_{AssetId})$

File: zip-0226.rst [hunk 2]
```diff
@@ -159,35 +159,63 @@ The Orchard Protocol uses a Homomorphic Pedersen Commitment [#protocol-concretev
...
+In order to verify the balance of the different Assets, the verifier MUST perform a similar process as for the Orchard protocol [#protocol-binding]_, with the addition of the burn information.
 
 For a total of :math:`n` Actions in a transfer, the prover MUST still sign the SIGHASH transaction hash using the binding signature key
 :math:`\mathsf{bsk} = \sum_{i=1}^{n} \mathsf{rcv}_{i}`.
 
-The verifier MUST compute
+The verifier MUST compute the value balance verification equation:
 
-.. math:: \mathsf{bvk} =(\sum_{i=1}^{n}  \mathsf{cv^{net}}_{i})  - \mathsf{ValueCommit_0^{OrchardZSA}(v^{balanceOrchard}, \mathcal{V}^{\mathsf{Orchard}})}
+.. math:: \mathsf{bvk} =(\sum_{i=1}^{n}  \mathsf{cv^{net}}_{i})  - \mathsf{ValueCommit_0^{OrchardZSA}(v^{balanceOrchard}, \mathcal{V}^{\mathsf{Orchard}})} - \sum_{k=1}^{L} \mathsf{ValueCommit_0^{OrchardZSA}}(\mathsf{v}_{k}, \mathsf{AssetBase}_{k})
```
Comment (PaulLaux) [inline zip-0226.rst:194]: Replace the running index `k` with `AssetId` to comply with previous definitions.

File: zip-0226.rst [hunk 3]
```diff
@@ -159,35 +159,63 @@ The Orchard Protocol uses a Homomorphic Pedersen Commitment [#protocol-concretev
...
+
+.. math:: \mathsf{assetBurn} = \{ (\mathsf{AssetBase},\mathsf{v^{AssetBase}})\ |\ \forall\ \mathsf{AssetBase}\ \textit{s.t.}\ \mathsf{v^{AssetBase}} \neq 0 \}
+
+We denote by :math:`L` the cardinality of the :math:`\mathsf{assetBurn}` set.
+
+Additional Consensus Rules
+``````````````````````````
+
+1. We require that :math:`\forall (\mathsf{AssetBase},\mathsf{v^{AssetBase}}) \in \mathsf{assetBurn}\ ,\ \mathsf{AssetBase} \neq \mathcal{V}^{\mathsf{Orchard}}`. That is, ZEC or TAZ is not allowed to be burnt.
+2. We require that for every :math:`\forall (\mathsf{AssetBase},\mathsf{v^{AssetBase}}) \in \mathsf{assetBurn}\ ,\ \mathsf{v^{AssetBase}} \neq 0`.
```
Comment (PaulLaux) [inline zip-0226.rst:179]: Adjust to comply with the other changes.

**Review by PaulLaux — COMMENTED** [review]
Good overall, some more adjustments are required.

For consistency:
- at the **Value Commitment** section, remove the supertext *Orchard* from
$AssetBase^{Orchard}$, (TBD across the entire document).

- Next, we have $AssetBase_{AssetId}$ , *AssetId* is an index, not to be confused with a constant.
change 
We define $\mathsf{AssetBase}^{\mathsf{Orchard}}_{\mathsf{ZEC}} :=\mathcal{V}^{\mathsf{Orchard}}$ so that the value commitment for ZEC notes is computed identically to the Orchard protocol deployed in NU5.
To be something similar to
For ZEC, we define $AssetBase :=\mathcal{V}^{\mathsf{Orchard}}$ (the index can not specify that it is zec so we need to add it into the text).

 - Scan entire document and the other zip to make sure all places have the same definitions.

File: zip-0226.rst (hunk 1 above)
Comment (vivek-arte) [reply zip-0226.rst:171]: Done

File: zip-0226.rst (hunk 2 above)
Comment (vivek-arte) [reply zip-0226.rst]: Done.

File: zip-0226.rst (hunk 3 above)
Comment (vivek-arte) [reply zip-0226.rst]: Done.

**Comment by vivek-arte** [comment]
Have made these adjustments in the ZIPs.

**Review by PaulLaux — APPROVED** [review]
Very nice.

**Pull request merged**